gandalf.js

```diff
@@ -1,4 +1,4 @@
-const capitalizedString = string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.